### ISSUE TITLE
Save state between restarts

### ISSUE THREAD

**Issue by macjl** [jhthorsen/homebridge-tasmota-aircon-http#4]
Hello,

I’ve found that at Homebridge restart, the AC devices provided by this plugin are set to off, with 20°C temperatures.

It’s not synchronized to the real state of AC, that could have been set to On, with different temperatures, or fan speed previously.

I think the plugin should save the last state at each commands, and restore it at launch. This would avoid this issue.

I don’t know how to do it, or if there are Homebridge function to do it. If you have an idea and can tell me where to search, I can try to submit a PR if you are ok with it.

**Comment by jhthorsen** [comment]
Sounds like an excellent addition! I looked at #6, which I think is a very good start. I'm curious if you have a reference Homebridge plugin that also use https://www.npmjs.com/package/node-persist ?

**Comment by jhthorsen** [comment]
I'm going to close this issue, since I'm confident #7 will get merged. Great work!

**Comment by jhthorsen** [comment]
Sorry, but I had to revert the commit, since it broke all the tests and was using an old version of node-persist.

**Comment by macjl** [comment]
Hello,
Did you see the update I made on the PR ? It now uses the last version of node-persist. But I did not look at the tests for now.

**Comment by jhthorsen** [comment]
Yes, I did: https://github.com/jhthorsen/homebridge-tasmota-aircon-http/pull/11#issuecomment-1139635704

**Comment by jhthorsen** [comment]
Fixed in #11.
